### ISSUE TITLE
pyarrow build failure.

### ISSUE THREAD

**Issue by heroxbd** [6-6-6/spark-overlay#5]
pyarrow 找不到已经安装了的 arrow 库。

[build.log](https://github.com/6-6-6/spark-overlay/files/6413427/build.log)

**Comment by heroxbd** [comment]
@littlewu2508

**Comment by heroxbd** [comment]
```
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:165 (message):
  Could NOT find Arrow (missing: ARROW_LIB_DIR) (found version "3.0.0")
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:458 (_FPHSA_FAILURE_MESSAGE)
  cmake_modules/FindArrow.cmake:419 (find_package_handle_standard_args)
  cmake_modules/FindArrowPython.cmake:46 (find_package)
  CMakeLists.txt:214 (find_package)
```

**Comment by heroxbd** [comment]
```
# emerge -pv apache-arrow

Calculating dependencies      ... done!                      
[ebuild   R   ~] dev-libs/apache-arrow-3.0.0::spark-overlay  USE="bzip2 lz4 parquet python zlib zstd" 0 KiB
```

**Comment by Leo3418** [comment]
Was this resolved by pull request #10?

**Comment by heroxbd** [comment]
Indeed!
